components/list/list-item-drag-drop-mixin.js

```diff
@@ -359,6 +368,85 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 		}));
 	}
 
+	_dispatchListItemsMove(sourceItems, targetItem, moveLocation, keyboardActive) {
```

Comment:
This `d2l-list-items-move` is the new event we want consumers to use.  They basically get the items being moved, the target item where they are being moved to, and the location with respect to that item - `above`, `below`, `nest`.